R/pkg/R/mllib.R

```diff
@@ -563,11 +574,12 @@ read.ml <- function(path) {
 #' \code{predict} to make predictions on new data, and \code{write.ml}/\code{read.ml} to
 #' save/load fitted models.
 #'
-#' @param data A SparkDataFrame for training
-#' @param formula A symbolic description of the model to be fitted. Currently only a few formula
+#' @param data a SparkDataFrame for training.
+#' @param formula a symbolic description of the model to be fitted. Currently only a few formula
 #'                operators are supported, including '~', ':', '+', and '-'.
 #'                Note that operator '.' is not supported currently
-#' @return \code{spark.survreg} returns a fitted AFT survival regression model
+#' @param ... additional argument(s) passed to the method.
```

Comment:
or document as `Currently not used.` like http://ugrad.stat.ubc.ca/R/library/e1071/html/predict.naiveBayes.html